/*umer testing merging with branch*/
int x[9];
int func(void){
    return 9;
}
void main(void){
    int hello;
    int num[3];

    hello = x[func()]; /*Should this work if num is null? */
   
}
/* make sure void x; gives error */